### PR TITLE
Update module.lua

### PR DESCRIPTION
**Pull request by juliiiocesar**

### CODE REVIEW

File: modules/__base__/accessories/client/module.lua [hunk 1]
```diff
@@ -130,22 +136,20 @@ module.OpenShopMenu = function(accessory)
 
   TriggerEvent('esx_skin:openRestrictedMenu', function(data, menu)
 
-    menu.close()
+    menu:destroy()
```
Comment (izio38) [inline modules/__base__/accessories/client/module.lua:139]: You just have to net menu to nil.

File: modules/__base__/accessories/client/module.lua [hunk 2]
```diff
@@ -179,14 +182,14 @@ module.OpenShopMenu = function(accessory)
       module.CurrentActionMsg = _U('accessories:press_access')
       module.CurrentActionData = {}
     end, function(data, menu)
-      menu.close()
+      menu:destroy()
```
Comment (izio38) [inline modules/__base__/accessories/client/module.lua:185]: You just have to net menu to nil.

File: modules/__base__/accessories/client/module.lua [hunk 3]
```diff
@@ -179,14 +182,14 @@ module.OpenShopMenu = function(accessory)
...
       module.CurrentAction = 'shop_menu'
       module.CurrentActionMsg = _U('accessories:press_access')
       module.CurrentActionData = {}
     end)
   end, function(data, menu)
 
-    menu.close()
+    menu:destroy()
```
Comment (izio38) [inline modules/__base__/accessories/client/module.lua:192]: You just have to net menu to nil.

File: modules/__base__/accessories/client/module.lua [hunk 4]
```diff
@@ -45,7 +46,7 @@ module.Init = function()
 
       on('esx:interact:enter:' .. key, function(data)
 
-      ESX.ShowHelpNotification(_U('accessories:press_access'))
+      utils.ui.showHelpNotification(_U('accessories:press_access'))
```
Comment (izio38) [inline modules/__base__/accessories/client/module.lua:49]: Indent is not correct here, could you add tabs to this block ?

**Comment by ArkSeyonet** [comment]
Accessorries is being completely reworked. Thanks for the contribution though.

**Pull request closed without merging**